BackofficeBundle/EventSubscriber/BoDirectionChildrenSubscriber.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+
+/**
+ * Class BoDirectionChildrenSubscriber
+ */
+class BoDirectionChildrenSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @param FormEvent $event
+     */
+    public function preSetData(FormEvent $event)
+    {
+        $form = $event->getForm();
+        $data = $event->getData();
+
+        if (null !== $data->getId()) {
```

Comment:
because when you create a node, the bo direction is the bo direction of the source (node or template)